src/main/java/com/cvsgo/repository/ReviewCustomRepository.java

```diff
@@ -9,4 +11,7 @@
 public interface ReviewCustomRepository {
     List<SearchReviewQueryDto> searchByFilter(User user, SearchReviewRequestDto request, Pageable pageable);
 
+    List<ReadReviewQueryDto> findAllByFilter(User loginUser, Long productId,
```

Comment:
현재 메서드명만 봐서는 `searchByFilter`과 하는 일이 어떻게 다른지 모르겠어서 **productId에 따른 review를 가져오는 함수**라는 것을 직관적으로 알 수 있게 함수명을 바꾸는것은 어떨까요?
```suggestion
    List<ReadReviewQueryDto> findByProductId(User loginUser, Long productId,
```

Reply:
searchByFilter도 메소드명을 나중에 리팩토링할 때 findAllByFilter로 바꾸려고 해서, 메소드명을 동일하게 오버로딩할 생각이었습니다. 그런데 채원님 말씀대로 ByProductId가 붙는 것이 좋을 것 같아요! findAllByFilterAndProductId는 혹시 어떤가요?

Reply:
구두로 논의한대로 findAllByProductIdAndFilter로 수정했습니다~